### PR TITLE
Fix --qlconfig-file option

### PR DESCRIPTION
**Pull request by aeisenberg**
Fixes an invalid CLI option being passed to `database init`.

### CODE REVIEW

File: src/codeql.test.ts [hunk 1]
```diff
@@ -1083,7 +1083,7 @@ test("passes a code scanning config BUT NOT a qlconfig to the CLI when CLI confi
 
   // should have passed a qlconfig file
   const hasQlconfigArg = args.some((arg: string) =>
-    arg.startsWith("--qlconfig=")
+    arg.startsWith("--qlconfig-file=")
   );
   t.false(hasQlconfigArg, "Should have injected a codescanning config");
```
Comment (henrymercer) [inline src/codeql.test.ts:1088]: This assertion is incorrect, for instance.

**Review by henrymercer — COMMENTED** [review]
There are a couple more bugs I fixed in the bundle update branch.  Would you consider cherry-picking [a6d3a44](https://github.com/github/codeql-action/pull/1561/commits/a6d3a44519505c57fd42bcc907af45b7776836bb), [100bd7b](https://github.com/github/codeql-action/pull/1561/commits/100bd7bbef94953be41fddd5afa784161cf2b4c5), and [eb40427](https://github.com/github/codeql-action/pull/1561/commits/eb40427b008b0874e104d0d25a575f86308ad9da)?

**Comment by henrymercer** [comment]
@angelapwen Would you mind taking a quick look?  It's Andrew's PR and my commits, so neither of us can review 😆

**Review by angelapwen — APPROVED** [review]
Changes look straightforward ✅

**Pull request merged**